dev/Pager/InteractionTests/PagerTests.cs

```diff
@@ -0,0 +1,960 @@
+
+                VerifyPageChangedEventOutput(4);
+                VerifyLastPageButtonIsEnabled(false);
+                VerifyLastPageButtonVisibility();
+
+                InputHelper.LeftClick(elements.GetFirstPageButton());
+
+                VerifyPageChangedEventOutput(0);
+                VerifyLastPageButtonIsEnabled();
+                VerifyLastPageButtonVisibility();
+
+                SetLastPageButtonVisibilityMode(ButtonVisibilityMode.HiddenOnEdge);
+
+                VerifyLastPageButtonVisibility();
+                VerifyLastPageButtonIsEnabled();
+
+                InputHelper.LeftClick(elements.GetNextPageButton());
+
+                VerifyPageChangedEventOutput(1);
+                VerifyLastPageButtonIsEnabled(true);
+                VerifyLastPageButtonVisibility();
+
+                InputHelper.LeftClick(elements.GetLastPageButton());
+
+                VerifyPageChangedEventOutput(4);
+                VerifyLastPageButtonIsEnabled(false);
+                VerifyLastPageButtonVisibility(Visibility.Collapsed);
+
+                InputHelper.LeftClick(elements.GetFirstPageButton());
+
+                VerifyPageChangedEventOutput(0);
+                VerifyLastPageButtonIsEnabled();
+                VerifyLastPageButtonVisibility();
+
+                SetLastPageButtonVisibilityMode(ButtonVisibilityMode.None);
+
+                VerifyLastPageButtonVisibility(Visibility.Collapsed);
+                VerifyLastPageButtonIsEnabled();
+
+                InputHelper.LeftClick(elements.GetNextPageButton());
+
+                VerifyPageChangedEventOutput(1);
+                VerifyLastPageButtonIsEnabled();
+                VerifyLastPageButtonVisibility(Visibility.Collapsed);
+
+                InputHelper.LeftClick(elements.GetNextPageButton());
+                VerifyPageChangedEventOutput(2);
+                InputHelper.LeftClick(elements.GetNextPageButton());
+                VerifyPageChangedEventOutput(3);
+                InputHelper.LeftClick(elements.GetNextPageButton());
+                VerifyPageChangedEventOutput(4);
+
+                VerifyLastPageButtonIsEnabled(false);
+                VerifyLastPageButtonVisibility(Visibility.Collapsed);
+
+                InputHelper.LeftClick(elements.GetFirstPageButton());
+
+                VerifyPageChangedEventOutput(0);
+                VerifyLastPageButtonIsEnabled();
+                VerifyLastPageButtonVisibility(Visibility.Collapsed);
+
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "A")]
+        public void AutoDisplayModeTest()
+        {
+            using (var setup = new TestSetupHelper("Pager Tests"))
+            {
+                elements = new PagerTestsPageElements();
+                SetAutoDisplayMode();
+                VerifyAutoDisplayMode();
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "A")]
+        public void NumberBoxDisplayModeTest()
```

Comment:
>NumberBoxDisplayModeTest [](start = 20, length = 24)

Is this test valuable? I think it is already covered by the NumberBoxDisplayChangingPageTest

Reply:
Removed duplicate test cases!